packages/blade/package.json

```diff
@@ -93,7 +93,7 @@
     "build:clean-theme-bundle": "rm -rf build/js-bundle-for-css",
     "react-native:get-stories": "sb-rn-get-stories --config-path=./.storybook/react-native && yarn prettier --write ./.storybook/react-native/storybook.requires.js",
     "react-native:storybook:android": "yarn react-native:get-stories && FRAMEWORK=REACT_NATIVE react-native run-android",
-    "react-native:storybook:ios": "yarn react-native:get-stories && FRAMEWORK=REACT_NATIVE react-native run-ios",
+    "react-native:storybook:ios": "yarn react-native:get-stories && FRAMEWORK=REACT_NATIVE react-native run-ios --simulator=\"iPhone 13\"",
```

Comment:
Not sure actually, might break but after pulling changes from master it suddenly started spinning up Iphone 8 on my machine. 

Since mostly blade devs will spin up RN storybook let us all use the same Iphone 13 simulator?

Reply:
It will break which is why I didn't add this by default.

Reply:
I use iPhone 14 pro :P

Reply:
I see, in that case how do we set default simulator on my end? It's currently opening iphone 8 by default

Reply:
Okay will revert this, will see if i can set a default.